apps/web/server/routers/viewer/eventTypes.tsx

```diff
@@ -266,7 +266,6 @@ export const eventTypesRouter = createProtectedRouter()
         users,
         id,
         hashedLink,
-        price,
```

Comment:
This was added previously, so it was being omitted on the data object to update.